itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/odo/CreateS2IComponentODOTest.java

```diff
@@ -43,7 +43,7 @@ public static void setupWorkspace() {
 	
 	@Test
 	public void testCreateComponent() {
-		createComponent(projectReq.getProjectName());
+		createComponent(projectReq.getProjectName(), "java", true);
```

Comment:
should be false here for s2iComponent
```suggestion
		createComponent(projectReq.getProjectName(), "java", false);
```